source/sessions/snapshot-sessions.rst

```diff
@@ -0,0 +1,358 @@
+================================
+Snapshot Reads Specification
+================================
+
+:Spec Title: Snapshot Reads Specification (See the registry of specs)
```

Comment:
I don't think we need to call out a registry of specs here
```suggestion
:Spec Title: Snapshot Reads Specification
```

Reply:
Not sure what's the convention here as this referral appears in `causal-consistency.rst` and `driver-sessions.rst`.
Removed for now.

Reply:
I believe those are leftovers from the templates. I'll give the specs a quick look to clean these up. Thanks for pointing out other instances.